.github/workflows/commit_message_checker.yml

```diff
@@ -0,0 +1,38 @@
+name: 'Commit Message Check'
+on: [push, pull_request]
```

Comment:
You need
```
# yamllint disable-line rule:truthy
```
before that line.
In YAML 1.1 `on` was considered a boolean, but GutHub Workflows don't use YAML 1.1

Reply:
Thanks for the info! Fixed